getting-started/templates/systemlink-values.yaml

```
@@ -508,6 +508,12 @@ fileingestion:
     ## S3 Port
     ##
     port: *minioPort
+  ## Configure rate limiting per user per replica over one second.
+  ##
+  rateLimits:
+    ## Upload file
+    ##
+    upload: 3
```

```suggestion
    uploadsPerSecond: "3"
```

Use quotes unless you really want to the default value to be interpreted as a number in YAML.

---

The quote function is a catchall to help catch customer mistakes. There's no harm in using quote with string values. Helm best practice is to quote values you expect to treat as a string. I'm not going to push back if you really think the integer is better.

---

well, I thought the **number** of files that can be uploaded over a certain period should be a **number**. But I'm also good either way. I'll think abou this